.github/workflows/autopkgtest.yml

```diff
@@ -45,7 +45,12 @@ jobs:
           pull-lp-source netplan.io
           cp -r netplan.io-*/debian .
           rm -r debian/patches/  # clear any distro patches
-          sed -i 's|rm debian/tmp/lib/netplan/generate|# DELETED|' debian/rules
+          # usrmerge-fix
+          mkdir debian/extra
```

Comment:
I believe this will start to fail when your changes get synced with Ubuntu but we can fix the CI again later.

Reply:
Yes. And I just uploaded 0.107.1-2 into Debian unstable. I'll use my next PR to adopt CI accordingly, once that landed.